recipe-server/normandy/recipes/api/v1/serializers.py

```diff
@@ -34,6 +58,19 @@ class Meta:
         ]
 
 
+class SignedActionSerializer(serializers.ModelSerializer):
+    signature = SignatureSerializer()
+    action = serializers.SerializerMethodField()
```

Comment:
No, because then it would look for a member called `action` on the Action instance, when instead we want the Action itself.